### DISCUSSION TITLE
How would I go on about debugging file-loader in webpack v4?

### DISCUSSION THREAD

**Discussion by runofthemillgeek** [webpack/webpack#12867] [Q&A]
Hi there. I was trying to debug an issue that stemmed from `file-loader` responding to `.mjs` files. I tried to debug this by setting `stats: "verbose"` in `devServer` and core webpack config along with other similar settings. I did get a detailed log but I couldn't find a line there that told me that `file-loader` touched this particular file, if that makes sense. I can see lines about `css-loader` and such but none about `file-loader`. Is this because it intentionally doesn't have debug logging or am I missing something?

Would appreciate in general, if there's any source about debugging webpack configs/loaders/plugins.

**Comment by alexander-akait** [comment]
What it the problem?

**Reply by runofthemillgeek** [reply]
I've managed to resolve it by excluding `.mjs` in the config. It was picking up `.mjs` files and producing them in the output which caused "export X not found" sort of errors. But I was just curious as to if there are any logs about which loaders got invoked to help debug issues like this.
